app/views/orders/index.html.erb

```diff
@@ -12,7 +11,7 @@
         </tr>
       </thead>
       <tbody>
-        <% @orders.each do |order| %>
+        <% @orders.reverse_each do |order| %>
```

Comment:
Instead of using `reverse_each` here, you can order the `@orders` variable in the orders_controller#index action, like so `@orders = current_company.orders.order(created_at: :desc)`
This